### PR TITLE
Add Layzr.js

### PR DESCRIPTION
**Pull request by irazasyed**
Project: https://github.com/callmecavs/layzr.js

### CODE REVIEW

File: files/layzr.js/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "layzr.js",
+  "repo": "callmecavs/layzr.js",
+  "files": {
+    "basePath": "dist/"
```
Comment (jimaek) [inline files/layzr.js/update.json]: can you add an `include: *.min.*` parameter(with correct formatting)? To make sure we host only the minified files.

**Comment by irazasyed** [comment]
@jimaek Updated as per requirements.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jimaek** [comment]
@irazasyed  Thank you

**Comment by irazasyed** [comment]
@jimaek No problem. Thanks for merging :+1:

**Pull request closed without merging**